controllers/appgroup_controller_testutils_test.go

```diff
@@ -1,15 +1,15 @@
-package controllers
+package controllers_test
```

Comment:
The file name is loooong. Any particular reason for switching it from testutils?

Reply:
I changed package to controllers_test. So, I need to change testutils.go to testutils_test.go. This might confuse someone to think that testutils_test.go contains tests for testutils.go. If you want, I can change this to testutils_test.go